app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -211,7 +211,8 @@ public BgGraphBuilder(Context context, long start, long end, int numValues, bool
             loaded_start = start;
             loaded_end = end;
             bgReadings = BgReading.latestForGraph(numValues, start, end);
-            if (DexCollectionType.getDexCollectionType() == DexCollectionType.LibreReceiver)
+            if (DexCollectionType.getDexCollectionType() == DexCollectionType.LibreReceiver
+            || (DexCollectionType.getDexCollectionType () == DexCollectionType.Follower && prefs.getBoolean("Libre2_showRawGraph",false)))
```

Comment:
Still two minor code formatting changes from previous to your commit. Please insert a space before `false` in `prefs.getBoolean("Libre2_showRawGraph",false)` at line 215 and 1323.